src/test/java/stringCalculator/OperatorTest.java

```diff
@@ -0,0 +1,57 @@
+package stringCalculator;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import stringCalculator.domain.Operator;
+
+class OperatorTest {
+
+    @Test
+    @DisplayName("operator를 정확하게 찾는지 확인한다.")
+    void test1() {
+        String input = "+";
+        Operator expectingResult = Operator.ADD;
+
+        assertThat(Operator.find(input)).isEqualTo(expectingResult);
```

Comment:
```suggestion
        assertThat(Operator.valueOf(input)).isEqualTo(expectingResult);
```

`Enum`을 사용할때는 `valueOf`이름을 많이 사용들 하시더라구요~ 😄